src/features/invoiceCard/InvoiceCardContainer.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { PureQueryOptions } from 'apollo-client';
+
+import Modal from '../../common/modal/Modal';
+import SendInvoiceForm from './sendInvoiceForm/SendInvoiceFormContainer';
+import EditForm from './editForm/EditForm';
+import InvoiceCard, { InvoiceCardProps } from './InvoiceCard';
+
+export interface InvoiceCardContainerProps extends Omit<InvoiceCardProps, 'sendInvoice' | 'editAdditionalServices'> {
+  customerEmail: string | null;
+  refetchQueries: PureQueryOptions[] | string[];
+}
+
+const InvoiceCardContainer = ({
+  customerEmail,
+  refetchQueries,
+  order,
+  ...invoiceCardProps
+}: InvoiceCardContainerProps) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [isSending, setIsSending] = useState(false);
```

Comment:
This state smells like a in progress spinner, should it be just `sendInvoiceFormOpen` ?